components/counter-bar/src/index.js

```diff
@@ -177,7 +178,10 @@ const CounterWrapper = styled(TotalWrapper, {
  * Custom colours on total
  * ```jsx
  * <CounterBar>
- *   <CounterBar.Total score={15} scoreColor="yellow" scoreBackgroundColor="pink">All counters</CounterBar.Total>
+ *   <CounterBar.Total
+ *    score={15}
+ *    scoreColor="yellow"
+ *    scoreBackgroundColor="pink">All counters</CounterBar.Total>
```

Comment:
if you're going to split the `CounterBar.Total` opening tag across multiple lines, then IMHO it's best to put the content and closing tags in separate lines to make the code more reasonable

plus also the indent should be kept consistent, i.e. attributes should go 2 characters in

i.e. this should be:
```
  *   <CounterBar.Total
  *     score={15}
  *     scoreColor="yellow"
  *     scoreBackgroundColor="pink"
  *   >
  *     All counters
  *   </CounterBar.Total>
```

Reply:
Thanks. This one _isn't_ an ESLint auto-fix!